Tests/PointFreeTests/__Snapshots__/DiscountsTests/testDiscounts_LoggedOut.1.Conn.txt

```diff
@@ -0,0 +1,2082 @@
+                          <h2 class="fg-black bold ts-d-r3 ts-m-r2 lh-2 light fg-gray650">
+                            <span class="content0">
+                              $17/mo
+                            </span>
+                            <span class="content1">
+                              $
+                              <span class="team-price" data-price="16">
+                                32
+                              </span>
+                              /
+                              mo
+                            </span>
+                          </h2>
+                        </div>
+                        <div class=" col-m col-m-12">
+                          <h6 class="fg-black bold ts-d-r0_875 ts-m-r0_75 lh-1 caps fg-gray650 inline">
+                            Monthly Plan
+                          </h6>
+                        </div>
+                      </div>
+                    </label>
+                  </div>
+                  <div class="bg-white col-m col-m-6">
+                    <label for="yearly" class="block m-ml3 m-mr3 m-mt3 m-mb3">
+                      <div style="flex-direction:column-reverse" class=" row">
+                        <input id="yearly" name="pricing[billing]" type="radio" value="yearly">
+                        <div class=" col-m col-m-12">
+                          <h2 class="fg-black bold ts-d-r3 ts-m-r2 lh-2 light fg-gray650">
+                            <span class="content0">
+                              $170/yr
+                            </span>
+                            <span class="content1">
+                              $
+                              <span class="team-price" data-price="160">
+                                320
+                              </span>
+                              /
+                              yr
+                            </span>
+                          </h2>
+                        </div>
+                        <div class=" col-m col-m-12">
+                          <h6 class="fg-black bold ts-d-r0_875 ts-m-r0_75 lh-1 caps fg-gray650 inline">
+                            Yearly Plan
+                          </h6>
+                        </div>
+                      </div>
+                    </label>
+                  </div>
+                  <div class="bg-white col-m col-m-12">
+                    <p class="content1 m-pb1 fg-gray400 fg-black normal h6 lh-4 w-100p center normal">
+                      20% off the Individual Monthly plan
+                    </p>
+                  </div>
+                </div>
+                <div class="bg-white row">
+                  <div class=" col-m col-m-12">
+                    <div class="m-pb3 m-pt2">
+                      <a href="/login?redirect=http://localhost:8080/pricing?expand%3Dtrue" class="medium cursor-pointer nowrap btn-normal rounded border-none text-decoration-none pf-link-white fg-white bg-black h5 m-pl2 m-pr2">
```

Comment:
this is the test snapshot for when we are logged out on the /discount/{orgname} url, and here we can see that the login redirect goes back to the pricing page with the expand=true flag.